server/routes/support/handlers/offenderAudit.ts

```diff
@@ -27,7 +27,7 @@ export default class OffenderAuditRoutes {
     res.render('pages/support/offenderAudit', {
       prisonerDetail: {
         id: nomsId,
-        name: convertToTitleCase(`${prisonerDetail.firstName} ${prisonerDetail.lastName}`),
+        name: (!!prisonerDetail && convertToTitleCase(`${prisonerDetail.firstName} ${prisonerDetail.lastName}`)) || '',
```

Comment:
I think these could be undefined still

Reply:
if no prisoner is found

Reply:
Yes, if it is undefined it will return ''(empty string instead of undefined undefined)